src/Wpf.Ui/Services/Internal/NavigationServiceActivator.cs

```diff
@@ -50,13 +53,112 @@ public static FrameworkElement CreateInstance(Type pageType, object dataContext)
             if (dataContextConstructor != null)
                 return dataContextConstructor.Invoke(new[] { dataContext }) as FrameworkElement;
         }
+#if NET48_OR_GREATER
```

Comment:
Code duplication, cannot it be like
`#if NET48_OR_GREATER || NETCOREAPP3_0_OR_GREATER`

Reply:
thanks for the review and the tip, corrected in the new commit